src/Eto/Forms/ThemedControls/ThemedDocumentControlHandler.cs

```diff
@@ -417,8 +584,11 @@ void Drawable_Paint(object sender, PaintEventArgs e)
 
 		posx = 0;
 
-		DrawTab(g, tabPrev, -1);
-		DrawTab(g, tabNext, -1);
+		if (allowNavigating)
```

Comment:
Those `<` and `>` buttons allow to switch to tabs that are not visible due to the width constraint. Would be nice to be able to scroll to those tabs or have some kind of an overflow menu (perhaps in a different PR)